lib/WebService/Stripe.pm

```diff
@@ -173,6 +173,12 @@ method cancel_transfer(Str $id, :$headers) {
     return $self->post("/v1/transfers/$id/cancel", undef, headers => $headers);
 }
 
+method create_reversal($xfer_id, HashRef $data = {}, HashRef $opts = {}) {
```

Comment:
I would really prefer if `$data` and `$opts` here were named params. Are you planning on adding the `$opts` param to all the methods? I'm not sure about adding it to just one method. Would you be open to holding off on adding `$opts` and the `opts_to_headers` thing until a future PR that adds it all at once to all the methods, and for now just passing `headers => { stripe_account => ... }`  as usual?